foodsaving/stores/serializers.py

```diff
@@ -229,3 +232,30 @@ def validate_weeks_in_advance(self, w):
         if w < 1:
             raise serializers.ValidationError(_('Set at least one week in advance'))
         return w
+
+
+class FeedbackSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = FeedbackModel
+        fields = ['id', 'given_by', 'about', 'weight', 'comment']
+        # update_fields = ['about', 'weight', 'comment']
+
+    def create(self, validated_data):
+        # validated_data['given_by'] = self.context['request'].user
```

Comment:
This should be the correct code. You probably need to configure the serializer to not expect `given_by` as input, by setting it as read_only.

It should work similar to here: https://github.com/yunity/foodsaving-backend/blob/95bee2e924dac544e84f6820c4071d6c2d2d2214/foodsaving/invitations/serializers.py#L14